qiita_pet/templates/study_ajax/artifact_file_selector.html

```diff
@@ -0,0 +1,186 @@
+<script type="text/javascript>">
+  /**
+   *
+   * Function that validates all the files lists
+   *
+   * This function triggers the sortupdate event in all the elements
+   * that are "checkeable"
+   */
+  function check_files_lists() {
+      // Reset the error counter
+      error_count = 0;
+      // initialize the error count to the number of file types
+      $(".checkeable").trigger("sortupdate");
```

Comment:
I just picked this up, but it seems a bit odd that the name of the class is `checkeable`, are you sure it's not `checkable`?